ppx_deriving_yojson.opam

```diff
@@ -16,10 +16,8 @@ depends: [
   "yojson" {>= "1.6.0" & < "2.0.0"}
   "result"
   "ppx_deriving" {>= "4.0" & < "5.0"}
```

Comment:
I think also the new version restriction for ppx_deriving is required, since older versions wouldn't work.